src/modules/Trip/trip.routes.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import TripController from './trip.controller';
+import ValidateRequestTrip from '../../middlewares/validateRequestTrip';
+import Auth from '../../middlewares/auth';
+
+
+const trip = express.Router();
+trip.post('/request', Auth.authorization, Auth.verifyToken, ValidateRequestTrip.validateRequestTripForm, TripController.tripRequest);
```

Comment:
I think authorization and verify token should be in the same function. Check @Steelze code for quick authentication

Reply:
i think your code will look more elegant if we try to remove chain of if-else statement. secondly a function should be doing a particular  thing not multiple things, and then chain them  as a middlewares to perform authorization

Reply:
Nice one.